src/app/components/action-button/index.html

```diff
@@ -31,4 +31,162 @@
     <sky-demo-page-code demoName="Action button"></sky-demo-page-code>
   </sky-demo-page-example>
 
+  <sky-demo-page-content sectionHeading="UX guidelines">
+		
+	<header><div class="sky-section-heading">Usage</div></header>
+		<p>Use action buttons when:</p>
+		<ul>
+			<li>The user has initiated a task or workflow</li>
+					<ul>
+							<li>This is often done via a page's primary action</li>
+					</ul>
+			<li>The task has between 2 and 6 methods of completion</li>
+					<ul>
+							<li>The user must choose exactly one option</li>
+							<li>If there are more relevant choices, consider a different <a routerLink="/design/guidelines/form-design" fragment="selection-inputs">selection input</a> and interaction</li>
+					</ul>
+			<li>The choices are system-defined, not user-defined</li>
+			<li>The choice of method will determine how the majority of the workflow will proceed</li>
+					<ul>
+							<li>This choice is made at or near the beginning of the task</li>
+					</ul>
+			<li>The outcome of end result will be largely the same, regardless of the method</li>
+		</ul>
+		<p>
+			Action buttons must represent a necessary step in any workflow they inhabit - they should not be used in ways where they can be skipped, avoided, or answered later. Present them in a way that isolates the choice they represent and removes other decisions, different tasks, or other content. Take advantage of the "details" line in the button to give users context or explain the differences between the options.
+		</p>	
+		
+	<header><div class="sky-subsection-heading">When to use action buttons</div></header>
+			
+  <stache-row>
+    <stache-column screenSmall="6">
+			<p>Use action buttons immediately after a user has initiated a task, to differentiate between different branches the task may follow.</p>
+		</stache-column>
+		<stache-column screenSmall="6">
+			<div class="screenshot screenshot-success">
+				<img src="~/assets/img/guidelines/actionbutton/action-buttons-in-modal.png" style="width: 100%;" />
+			</div>
+		</stache-column>
+	</stache-row>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            Use action buttons on a page or in a dialogue, but only when a task is underway and the choice about how to complete the task is the only choice relevant to the user. Do not display any other inputs or choices until the user picks an option. When the user picks an option, hide the action buttons (or close the modal).
```

Comment:
All of the plural changes and others sound good.

We definitely don't want confusion on this side note, though, so I'm open to suggestions for avoiding it. One path is chosen from several possible paths. If there's a guilty line suggesting otherwise, let's take a closer look.

Reply:
I think the confusion was with the item above. I've provided a suggested rewrite of that. If we update there, I think we're fine here.